styles.css

```diff
@@ -1,10 +1,8 @@
 img{
- 
```

Comment:
- [ ] Please try to be more professional in your comments and add solid reasons why removing the spacing or indentation is important also it seems like you are ordering, your comment should be clear, polite and professional. Kindly look at the point below, which will help you perform code reviews very well.
   - Kindly remove the spacing or indentation because it actually makes your code looks messy and makes your code base setup unprofessional.   

## Your comment

>  Also to add , it will be best to remove spacing that you don't require